### ISSUE TITLE
similarity calculate 

### ISSUE THREAD

**Issue by shengdoupi** [yao8839836/text_gcn#84]
Hi，is it possible that I use text_gcn to calculate similarity between two docs ?

**Comment by yao8839836** [comment]
@shengdoupi 

Yes, After obtaining document embeddings (train.py, line 162-163). You can compute similarity scores with cosine function.
